Error: Cannot find module './Instrumentation.js' when running frontend service for local development

# Bug Report

Which version of the demo you are using?

main (latest)

## Symptom

I'm trying to follow the README for running the frontend service locally. After running

docker compose run --service-ports -e NODE_ENV=development --volume $(pwd)/src/frontend:/app --volume $(pwd)/pb:/app/pb frontend sh

I get this error:

Error: Cannot find module './Instrumentation.js'
Require stack:
- internal/preload
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:956:15)
    at Function.Module._load (node:internal/modules/cjs/loader:804:27)
    at Module.require (node:internal/modules/cjs/loader:1028:19)
    at Module._preloadModules (node:internal/modules/cjs/loader:1303:12)
    at loadPreloadModules (node:internal/bootstrap/pre_execution:583:5)
    at prepareMainThreadExecution (node:internal/bootstrap/pre_execution:95:3)
    at node:internal/main/run_main_module:7:1 {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ 'internal/preload' ]
}

It looks like npm start is being run (per package.json) as opposed to npm dev. Am I doing something wrong here?

## Reproduce

Just follow the instructions for running the frontend service locally in the README file.

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -197,6 +197,7 @@ services:
       - currencyservice
       - otelcol
       - productcatalogservice
+      - quoteservice
       - recommendationservice
       - shippingservice
     logging: *logging

--- a/src/frontend/Dockerfile
+++ b/src/frontend/Dockerfile
@@ -39,4 +39,4 @@ USER nextjs
 ENV PORT 8080
 EXPOSE ${PORT}
 
-ENTRYPOINT ["npm", "start"]
+ENTRYPOINT npm start

--- a/src/frontend/README.md
+++ b/src/frontend/README.md
@@ -16,8 +16,7 @@ frontend client by going to <http://localhost:8080/>.
 Currently, the easiest way to run the frontend for local development is to execute
 
 ```shell
-  docker compose run --service-ports -e NODE_ENV=development
-  --volume $(pwd)/src/frontend:/app --volume $(pwd)/pb:/app/pb frontend sh
+docker compose run --service-ports -e NODE_ENV=development --volume $(pwd)/src/frontend:/app --volume $(pwd)/pb:/app/pb --user node --entrypoint sh frontend
 ```
 
 from the root folder.

